### ISSUE TITLE
【Try】1-GPU pretrain with big learning rate for 100W-step, then 1-GPU pretrain with small learning rate for another 100W-step.

### ISSUE THREAD

**Issue by guotong1988** [guotong1988/BERT-GPU#28]
compare to 4-GPU for downstream tasks.
